src/shared/clients/ssmClient.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AWSError, SSM } from 'aws-sdk'
+import globals from '../extensionGlobals'
+import { PromiseResult } from 'aws-sdk/lib/request'
+import { getLogger } from '../logger/logger'
+
+export class DefaultSsmClient {
```

Comment:
the DefaultXX naming pattern is left over from old code where interfaces were heavily used. If there is no `SsmClient` interface then we don't need "Default"
```suggestion
export class SsmClient {
```